web/packages/teleport/src/config.ts

```diff
@@ -95,6 +95,7 @@ const cfg = {
     clusters: '/web/clusters',
     trustedClusters: '/web/trust',
     audit: '/web/cluster/:clusterId/audit',
+    resources: '/web/cluster/:clusterId/resources',
```

Comment:
If I try to navigate to this path I just get a 404 error. Am I doing something wrong?

Reply:
Oh, sorry, forgot to tell you: I decided to hide it behind a `localStorage` setting until we bring the feature set on par with other views. Set `grv_teleport_unified_resources_enabled` in local storage to `true`.